cmd/kots/cli/pull.go

```diff
@@ -53,6 +53,7 @@ func PullCmd() *cobra.Command {
 					Username:  v.GetString("registry-username"),
 					Password:  v.GetString("registry-password"),
 				},
+				IsOpenShift:        v.GetBool("is-openshift"),
```

Comment:
is this the best way when its from the cli?

Reply:
is this in case the `IsOpenShift` function doesn't work?

Reply:
i would think you can get this info automatically